services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -636,10 +663,10 @@ testFindNonProvisionedUserNoIdP = do
     liftIO $ userManagedBy brigUser `shouldBe` ManagedByWire
     liftIO $ userEmail brigUser `shouldSatisfy` isJust
 
-  byHandle <- listUsers tok (Just (filterBy "userName" (fromHandle handle)))
   byExternalId <- listUsers tok (Just (filterBy "externalId" (fromEmail email)))
+  byHandle <- listUsers tok (Just (filterBy "userName" (fromHandle handle)))
```

Comment:
why?

Reply:
I just wanted to make sure that the "migration" happened also when looking up by `externalId` and not just by `userName`; when I first looked at the code I was a bit confused and thought: "oh, this works because we migrate the user _first_ by `userName`"

Reply:
ah, makes sense.  but in that case, we should just test both independently, no?  i'll see if i can rewrite it some more to make this clearer.